packages/components/src/components/cookies-consent.tsx

```diff
@@ -1,10 +1,11 @@
 'use client';
 
-import { useState } from 'react';
+import { ComponentProps, useState } from 'react';
 import { cn } from '../cn';
 import { CallToAction } from './call-to-action';
 
-export type CookiesConsentProps = React.HTMLAttributes<HTMLElement>;
+export type CookiesConsentProps = ComponentProps<'div'>;
```

Comment:
I wanted to comment about `ComponentPropsWithoutRef` but I think it's gonna be obsolete soon when React starts passing refs automatically, so `ComponentProps` is good.

<img width="775" alt="image" src="https://github.com/user-attachments/assets/21d668b9-9a3c-49e5-b91e-f10052f5902e">

Reply:
yeah, good catch, forget to use ...wtihoutProps